### PR TITLE
View convenience and searchable protocol

### PR DESCRIPTION
**Pull request by ndis1**
swift convenience methods. can't be put in a release until https://github.com/CocoaPods/CocoaPods/pull/4476 is in a release. till then we can use cocoapods dev off the master branch https://guides.cocoapods.org/using/unreleased-features

### CODE REVIEW

**Comment by ndis1** [comment]
this also drops support for ios 7.0

File: Pod/Classes/LayoutConvenience.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//  ConstraintConvenience.swift
+//  Pods
+//
+//  Created by Nick DiStefano on 12/11/15.
+//
+//
+
+import Foundation
+
+public func activateVFL(format: String, options: NSLayoutFormatOptions = [], metrics: [String : AnyObject]? = nil, views: [String : AnyObject]) {
```
Comment (codecaffeine) [inline Pod/Classes/LayoutConvenience.swift]: A general question: This function really just saves one line of code but it adds complexity by adding more grammar. Do we want stuff like this?

Comment (ndis1) [inline Pod/Classes/LayoutConvenience.swift]: yes, I want it 
it saves up to 3 lines of code per VFL

File: Pod/Classes/LayoutConvenience.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            format,
+            options: options,
+            metrics: metrics,
+            views: views
+        )
+    )
+}
+
+public extension UIView {
+    public func addCodelayoutView(subview: UIView) {
```
Comment (codecaffeine) [inline Pod/Classes/LayoutConvenience.swift]: Not crazy about his naming.

File: Pod/Classes/LayoutConvenience.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        subview.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(subview)
+    }
+}
```
Comment (codecaffeine) [inline Pod/Classes/LayoutConvenience.swift]: add newline please

File: Pod/Classes/SearchableProtocol.swift [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        attributes.displayName = infoProvider.searchTitle()
+        
+        attributes.relatedUniqueIdentifier = infoProvider.searchURL()
+        attributes.contentDescription = infoProvider.contentDescription()
+        
+        attributes.keywords = Array(infoProvider.keywords())
+        
+        return attributes
+    }
+}
```
Comment (codecaffeine) [inline Pod/Classes/SearchableProtocol.swift]: add newline please

File: Pod/Classes/SearchableProtocol.swift [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    var userActivityEligibility: UserActivityEligibility { get }
+    func searchURL() -> String
+    func searchTitle() -> String
+    func keywords() -> Set<String>
+    func contentDescription() -> String
+}
+
+@available(iOS 9.0, *)
+protocol Searchable {
+    func configActivity(searchActivity: NSUserActivity, infoProvider: SearchInfoProvider) -> NSUserActivity
```
Comment (codecaffeine) [inline Pod/Classes/SearchableProtocol.swift]: `configureActivity`?

**Pull request merged**